garrysmod/gamemodes/terrortown/gamemode/gamemsg.lua

```diff
@@ -235,6 +235,18 @@ local function TraitorGlobalVoice(ply, cmd, args)
 end
 concommand.Add("tvog", TraitorGlobalVoice)
 
+-- from shared.lua
+-- MUTE_NONE = 0
+-- MUTE_TERROR = 1
+-- MUTE_ALL = 2
+-- MUTE_SPEC = 1002
+local MuteModes = {
+   [0] = "mute_off",
+   [1] = "mute_living",
+   [2] = "mute_all",
+   [1002] = "mute_specs"
+}

```

Comment:
Wouldn't it be possible to use MUTE_NONE etc as the indexes here directly? `[MUTE_NONE] = "mute_off"` and so on.

Reply:
Oh, yes, I've made a mistake when testing. I didn't use `[]` around the indexes, and using those indexes was returning an error. But I just tested it and it works fine. Will update it

Reply:
I also thought about adding translations to the last words. But it ended up being a bit more complicated, since the translation functions only work client-side. Although I could "fetch" a translation by storing `LANG.Msg()` in a variable, which felt pretty weird and stupid for me lmao

What do you think about it?

Reply:
I think the way to do that would be to add the player's death type to the `TTT_LastWordsMsg`, and then do the interpretation (with `LastWordContext[ply.death_type]` etc) in the clientside handling (in cl_voice). Then retrieving a translation is trivial.

Of course, last words is off by default and probably quite rare in practice (voicechat is more popular, it's rare to die while typing, etc) so maybe it's not worth the effort.

Reply:
It doesn't sound like a bad idea for me. Actually could be a good project to practice some stuff.

By the way, could you do something about the TTT translation to move it to Crowdin? Since you don't see problems with that change, maybe you could ask the folks at Facepunch to implement it?